cmd/pebble/cmd_boot_firmware.go

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) 2022 Canonical Ltd
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 as
+// published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"syscall"
+	"time"
+
+	"github.com/jessevdk/go-flags"
+
+	"github.com/canonical/pebble/client"
+	. "github.com/canonical/pebble/cmd"
+	"github.com/canonical/pebble/internal/boot"
+	"github.com/canonical/pebble/internal/daemon"
+	"github.com/canonical/pebble/internal/logger"
+	"github.com/canonical/pebble/internal/plan"
+)
+
+type cmdBootFirmware struct {
+	clientMixin
+
+	Force   bool `long:"force"`
+	Verbose bool `short:"v" long:"verbose"`
+}
+
+var bootFirmwareDescs = map[string]string{
+	"force":   `Skip all checks`,
```

Comment:
I think it can be useful for debugging purposes. Sometimes I find myself booting a firmware to a shell and run pebble interactively from there after inspecting the environment variables or some aspects from the live system.